velox/exec/tests/SortBufferTest.cpp

```diff
@@ -97,6 +97,7 @@ TEST_F(SortBufferTest, singleKey) {
 
   // Specifies the sort columns ["c1"].
   sortColumnIndices_ = {1};
+  uint32_t numSpillRuns;
```

Comment:
This can be a test class member and reset in each test iteration if we have more than test iterations in a test case.